lib/tasks/scripts/chennai_hierarchy_cleanup.rb

```diff
@@ -0,0 +1,52 @@
+class ChennaiHierarchyCleanup
+  def self.call
+    raise "Cannot run this outside India production" unless CountryConfig.current[:name] == "India" && SimpleServer.env.production?
+
+    new.call
+  end
+
+  def initialize
+    @state = Region.state_regions.find_by_name("Tamil Nadu")
+    @new_district = Region.find_by_name("Chennai")
+    raise "Create Chennai district from dashboard before running" if @new_district.blank?
+
+    @old_districts = @state.district_regions.where.not(name: @new_district.name)
```

Comment:
This assumes that Chennai's the only place in TN. Can we make it a bit smarter by grepping for `Chennai...`?

If that's awkward, I'd even prefer to see a flat list of Chennai zone names here, instead of slurping in all TN districts.